modules/network/arbitrum/services.ts

```diff
@@ -0,0 +1,35 @@
+import { tokenService } from '../../token/token.service';
+import { PhantomStableAprService } from '../../pool/lib/apr-data-sources/phantom-stable-apr.service';
+import { BoostedPoolAprService } from '../../pool/lib/apr-data-sources/boosted-pool-apr.service';
+import { SwapFeeAprService } from '../../pool/lib/apr-data-sources/swap-fee-apr.service';
+import { GaugeAprService } from '../../pool/lib/apr-data-sources/ve-bal-gauge-apr.service';
+import { GaugeStakingService } from '../../pool/lib/staking/gauge-staking.service';
+import { BptPriceHandlerService } from '../../token/lib/token-price-handlers/bpt-price-handler.service';
+import { LinearWrappedTokenPriceHandlerService } from '../../token/lib/token-price-handlers/linear-wrapped-token-price-handler.service';
+import { SwapsPriceHandlerService } from '../../token/lib/token-price-handlers/swaps-price-handler.service';
+import { UserSyncGaugeBalanceService } from '../../user/lib/user-sync-gauge-balance.service';
+import { GithubContentService } from '../../content/github-content.service';
+import { gaugeSubgraphService } from '../../subgraphs/gauge-subgraph/gauge-subgraph.service';
+import { CoingeckoPriceHandlerService } from '../../token/lib/token-price-handlers/coingecko-price-handler.service';
+import { coingeckoService } from '../../coingecko/coingecko.service';
+import { IbTokensAprService } from '../../pool/lib/apr-data-sources/ib-tokens-apr.service';
+import { arbitrumNetworkData as data } from './data';
+
+export const arbitrumCreateServices = () => ({
```

Comment:
they would need to be typed to NetworkService I guess?

Reply:
i'm thinking how helpful that would be, since it's a static config adding a type only helps when someone forgets a required service, but even then it would get picked up by inference.